### ISSUE TITLE
Image printing 

### ISSUE THREAD

**Issue by rbabler** [alfonsovinti/cordova-plugin-bixolon-print#7]
Hi Alfonso,

Is there a plan to extend your Plugin to printing images?

Thanks

Best Regards
Rene

**Comment by pietervw** [comment]
Hi there @alfonsovinti and @rbabler.

Did you manage to get a solution to image printing?

**Comment by alfonsovinti** [comment]
Yes I will, but not in this moment. I planned to get this after the release of another printing plugin because I would like to merge in a same api the tow different printers.

I hope to start working on it as soon as possible.

Regards,

Alfonso

**Comment by pietervw** [comment]
@alfonsovinti  thank you for the feedback.

I did manage to get the plugin to print images.

I'll do a fork at a later stage to make the code available for everyone?

Regards,

**Comment by alfonsovinti** [comment]
That's great!! 

Thanks a lot for your contribution.

 Looking forward to hear from you soon

Regards

**Comment by pietervw** [comment]
Thanks for your work Alfonso.

I've forked it [here](https://github.com/pietervw/cordova-plugin-bixolon-print)

**Comment by lucav90** [comment]
Hi,

i have insert your fork into alfonso vinti plugin to print an image (printBitmap).

I have this error:


8 D/BixolonPrint: BixolonPrint.execute_START
12-11 21:36:18.000 18141-18141/com.i I/BixolonPrint: action: printBitmap
12-11 21:36:18.000 18141-18141/com.i D/BixolonPrint: Invalid action : printBitmap passed
12-11 21:36:18.000 18141-18141/com. D/BixolonPrint: BixolonPrint.execute_END
12-11 21:36:18.000 18141-18141/com.i W/CordovaPlugin: Attempted to send a second callback for ID: BixolonPrint944432401
Result was: "Invalid action"

How can i solve?

**Comment by sharma-open-source** [comment]
https://github.com/diegorquera/cordova-plugin-bixolon-printer/pull/2
